containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/mariadb-client-install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script is used to install a matching mariadb-client
+# in ddev-webserver
+# It should be called with the appropriate mariadb version as an argument
+# This script is intended to be run with root privileges (normally in a docker build phase)
+
+set -eu -o pipefail
+
+DDEV_DATABASE_FAMILY=${DDEV_DATABASE%:*}
+if [ "${DDEV_DATABASE_FAMILY}" != "mariadb" ]; then
+  echo "This script is to be used only with a project using mariadb" && exit 1
+fi
+MARIADB_VERSION=${DDEV_DATABASE#*:}
+# Add all required MariaDB versions here
+if [ "${MARIADB_VERSION}" != "11.4" ]; then
+  echo "This script is not intended to run with mariadb:${MARIADB_VERSION}" && exit 0
```

Comment:
Is this not an error? If it's an error, why `exit 0`? BTW, just `exit` should be same as `exit 0` but nothing wrong with exit 0.

Reply:
You're right, I thought about that, but then forgot. I will push a new image to replace the exit code.